JS/index.js

```diff
@@ -16,19 +16,27 @@ let waves = 1000;
 let numberWaves = 0;
 let lastShotTime = 0;
 
-function suus() {
+let runNum = 1;
+let enemyNum = 1;
 
+
+
+function getBattlefield(){
+    return  document.getElementById("outside-spaceship")
 }
 
 async function newGame() {

```

Comment:
You did not address the issue where I've mentioned this function.

Reply:
what do you mean?